crypto/s2n_hmac.c

```diff
@@ -142,25 +142,11 @@ int s2n_hmac_new(struct s2n_hmac_state *state)
     return 0;
 }
 
-int s2n_hmac_init(struct s2n_hmac_state *state, s2n_hmac_algorithm alg, const void *key, uint32_t klen)
+static int s2n_tls_hmac_init(struct s2n_hmac_state *state, s2n_hmac_algorithm alg, const void *key, uint32_t klen)
```

Comment:
Can you move this to immediately follow `s2n_sslv3_mac_init()` or `s2n_sslv3_mac_digest()` to keep each of the static functions together?

Reply:
Willdo